DOC-2029: Document new editable root APIs and options (#2757)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/editable_root.adoc

[[editable_root]]
== `+editable_root+`

> This option enables you to specify that the initial editable state of the root of {productname}. The root could still contain child elements with `contenteditable="true"` and those
> would remain editable even if you set this option to `false`.

Reviewer: ```suggestion
This option enables you to specify that the initial editable state of the root of {productname}.
Child elements with the `contenteditable="true"` attribute will still be editable when this option is `false`.
```
